cardano-submit-api/testing-tx-submit.md

```diff
@@ -0,0 +1,122 @@
+# Testing cardano-submit-api
+
+Setting this up for testing and for actual use on a real network.
+
+
+### Pre-requisites
+
+You will need some a cardano network with payment address and keys.  This may be on `mainnet`, official `testnet`
+or a testnet that you've set up yourself.  We will assume these files are in a direction called `playground`:
+
+* `user-1-payment.addr` - User 1 payment address.  This address must have sufficient funds.
+* `user-1-payment.vkey` - User 1 verification key.
+* `user-2-payment.addr` - User 2 payment address.
+* `magic.flag` - The network magic flag.  This will be `--testnet-magic <magic>` or `--mainnet`
+  dependening on the network you are using.
+* `node.socket` - The socket file for your network, or a symlink to that socket file.  If you
+  have a node running (for example, `mainnet` or `testnet` Daedalus), you can the socket file
+  using `ps aux | grep cardano-node` and looking for the `--node-socket`.
+
+You will also need to have `yj` installed.  This can be done via `brew`, `apt-get` or `snap`.
+
+### Install and run the cardano-submit-api
+
+Install the `cardani-submit-api`:
+
+```bash
+cardano-node $ cabal install cardano-submit-api --overwrite-policy=always
+```
+
+Install the `cardani-cli`:
+
+```bash
+cardano-node $ cabal install cardano-cli --overwrite-policy=always
+```
+
+Copy the configuration yaml to your `playground` and tweak it:
+
+```bash
+playground $ cat <configuration-yaml-file> | yj -jy > submit-api-config.yaml
+playground $ cat >> submit-api-config.yaml <<EOF
+EnableLogMetrics: False
+EnableLogging: True
+EOF
+```
+
+Then run the `cardani-submit-api` against your network:
+
+```bash
+playground $ cardano-submit-api --config submit-api-config.yaml --socket-path node.socket --port 8090
```

Comment:
Forgot to include `--testnet-magic xx` here

Reply:
Fixed!